uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/shared/controllers/request.controller.js

```diff
@@ -767,10 +790,11 @@ define([
      *
      * @return {Promise}
      */
-    function loadLoggedInContactId () {
-      return Session.get().then(function (value) {
```

Comment:
since we now don't use Session, can you please remove any reference to it from this file?

Reply:
Hm... I wonder why linter did not notice this.

Reply:
Done.

Reply:
The linter doesn't care if you don't use one of the parameters in your function. I guess it does that in case you need to use the third parameter, but not the second one.